lib/utils.js

```diff
@@ -29,7 +25,7 @@ function timeStringToMs(timeString, now) {
 
 function duringWorkingHoursOrNotSet(config, now) {
     const {workingHours} = config;
-    if (workingHours == null || nullOrEmpty(workingHours.from) || nullOrEmpty(workingHours.to)) {
+    if (!workingHours?.from || !workingHours?.to) {
```

Comment:
I'd consider checking for null or empty by doing this truthy/falsy check relatively unsafe for reason mentioned above. WDYT?